### ISSUE TITLE
Support both landscape modes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#34]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


Please use labels and text to provide additional information.


```

Original issue reported on code.google.com by `ideal2i...@gmail.com` on 26 Apr 2013 at 8:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `ideal2i...@gmail.com` on 26 Apr 2013 at 8:38
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Included in 0.9.7
```

Original comment by `ideal2i...@gmail.com` on 5 May 2013 at 9:53
- Changed state: **Done**
